src/explore-education-statistics-admin/src/pages/publication/components/ReleaseContributorPermissions.tsx

```diff
@@ -31,7 +31,7 @@ const ReleaseContributorPermissions = ({
               {contributors.map(contributor => (
                 <tr key={contributor.userId}>
                   <td className="govuk-!-width-one-half">
-                    {contributor.userDisplayName}
+                    {contributor.userDisplayName} ({contributor.userEmail})
```

Comment:
Would write this line as a single template string so that it's easier to find this text in tests